yabt/config.py

```diff
@@ -121,6 +121,10 @@ def get_build_file_path(self, build_module) -> str:
                    (BUILD_PROJ_FILE if '' == build_module
                     else self.build_file_name))
 
+    def get_go_packages_path(self) -> str:
+        return os.path.join(
+            self.project_root, self.builders_workspace_dir, 'go')
```

Comment:
for cache we use `.cache`, so maybe `.go` for consistency? don't feel strongly about this.